rs/orchestrator/src/upgrade.rs

```diff
@@ -369,6 +379,18 @@ impl Upgrade {
         )
         .map_err(OrchestratorError::UpgradeError)?;
         info!(self.logger, "Subnet state removed");
+
+        let instant = Instant::now();
+        sync_and_trim_fs(&self.logger)
+            .await
+            .map_err(OrchestratorError::UpgradeError)?;
+        let elapsed = instant.elapsed().as_millis();
```

Comment:
(TIL, you can ignore this) I was curious if using Instant across await points is correct in cases where the task is moved to another thread. Apparently on modern CPUs, TSCs are synchronized across cores and sockets, so this should work fine. Unfortunately the documentation for `CLOCK_MONOTONIC` is not clear about this.

Reply:
That's good to know, thanks for looking into it! 👍